demos/mcmcp/experiment.py

```diff
@@ -170,6 +68,7 @@ class Perturbation(Transformation):
 
 @extra_routes.route("/choice/<int:node_id>/<int:choice>", methods=["POST"])
 def choice(node_id, choice):
+    from sources import Agent
```

Comment:
The MCMCP demo has several kinds of custom objects: `Info`, `Transformation`,     `Source`, and `Agent`, so I don't think putting these all in a `sources.py` file is appropriate. I think there are two possible fixes: one is to separate these into four separate files; the other is to rename `sources.py` to something more generic like `extra_models`.